style.css

```diff
@@ -391,3 +406,166 @@ form > button:active {
   color: white;
   background: #2230d2;
 }
+
+@media screen and (min-width: 728px) {
+  #desk-list {
+    display: flex;
+    min-width: 200px;
+  }
+
+  #Union-menu {
+    display: none;
+  }
+
+  h1 {
+    font-size: 44px;
+    max-width: 700px;
+    font-family: var(--mainF);
+    font-weight: 700px;
+  }
+
+  main {
+    padding: 250px 250px 250px 250px;
+  }
+
+  p {
+    font-size: 20px;
+  }
+
+  #projects {
+    padding: 50px 50px;
+    align-self: center;
+  }
+
+  .card {
+    display: none;
+  }
+
+  .card-desk {
+    display: flex;
+    align-self: center;
+    background: #fff;
+    border: 2px solid #dfe1e6;
+    border-radius: 16px;
+    box-shadow: 0 0 0.5px 0.25px inset #c8c6c6;
+    margin-bottom: 50px;
+    transition: all ease-in-out 0.2s;
+    padding-left: 24px;
+    padding-right: 24px;
+    padding-top: 50px;
+    padding-bottom: 50px;
+    max-width: calc(100% - 100px);
+    flex-direction: row;
+  }
+
+  .details > img {
+    width: 81px;
+    height: 81px;
+    transform: rotate(-40.54deg);
+    padding-right: 150px;
+  }
+
+  .card-desk > div > img {
+    width: 100%;
+    transition: all ease-in-out 0.2s;
+  }
+
+  .card-desk:hover img {
+    transform: scale(1.05);
+  }
+
+  .card-desk:hover {
+    border: 2px solid #a7aeff;
+  }
+
+  .card-desk:hover .btn-action {
+    color: white;
+    background: #6070ff;
+    box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+  }
```

Comment:
- [ ] Good job adding some template projects in your portfolio :clap: , however , i've noticed that when you're hovering on the card the `image` is transformed and also the `btn-action` is changing :-1: . You should separate those feature so that when you're hovering on the image it gets transformed  and beside when you do it on the button it gets changed also separately :+1:  
![Screenshot from 2023-01-26 15-45-52](https://user-images.githubusercontent.com/80458199/214851946-3ed3b64b-4b12-4cc0-b102-6924b1a03ddf.png)